content/language/word-list.md

```diff
@@ -96,3 +96,29 @@ While master in and of itself is potentially neutral, the propensity in which it
 The above, as well as:
 
 * [github/renaming](https://github.com/github/renaming)
+
+## Bulletproof
+
+**Definition:** 
+Bulletproof is typically used in the context of cybersecurity to describe systems or applications that are relatively free from vulnerabilities.
+Bulletproof is also sometimes used in marketing communication when comparing products or services to those offered by competitors.
+
+**Proposed alternatives:** 
+
+In the cybersecurity context:
+* Secure
+* Safe
+* Protected
+
+In the marketing context:
+* Robust
+
+**Reasoning:** 
+
+The word bulletproof is associated with gun violence, especially in the USA. Some individuals find the term especially threatening or offensive.
```

Comment:
I propose to remove `, especially in the USA`. I feel it's like targeting the USA. Instead, we could possibly rephrase the statement as 

`The word bulletproof is associated with gun violence and it is militaristic.`

Reply:
> I propose to remove `, especially in the USA`. I feel it's like targeting the USA. Instead, we could possibly rephrase the statement as
> 
> `The word bulletproof is associated with gun violence and it is militaristic.`

```suggestion
The word bulletproof is associated with gun violence.
```

I recommend we set aside `miiltaristic` as a quality to judge a phrase by. It is enough in this and all cases to associate to terms of violence, and stop there. Otherwise, we are opening to a debate on the existence of the military in the world. That is an important discussion to have but I believe it goes far beyond the scope of Inclusive Naming Initiative.

A rule our word group at Red Hat has defined is to use military and martial terms strictly in context of the military. Thus, the phrase "let's take that hill" implies violence but could be done without, yet it is ultimately a term related to warfare and should be left to that domain.